src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -423,14 +423,14 @@ public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkRevo
         [HttpPut("{id}/restore")]
         public async Task RestoreAsync(Guid orgId, Guid id)
         {
-            await RestoreOrRevokeUserAsync(orgId, id, _organizationService.RestoreUserAsync);
+            await RestoreOrRevokeUserAsync(orgId, id, (orgUser, userId) => _organizationService.RestoreUserAsync(orgUser, userId, _userService));
         }
 
         [HttpPatch("restore")]
         [HttpPut("restore")]
         public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkRestoreAsync(Guid orgId, [FromBody] OrganizationUserBulkRequestModel model)
         {
-            return await RestoreOrRevokeUsersAsync(orgId, model, _organizationService.RestoreUsersAsync);
+            return await RestoreOrRevokeUsersAsync(orgId, model, (orgId, orgUserIds, restoringUserId) => _organizationService.RestoreUsersAsync(orgId, orgUserIds, restoringUserId, _userService));
```

Comment:
The issue is that `RestoreUsersAsync` takes `userService` as an argument, but `RevokeUsersAsync` (the other `statusAction`) does not. We can't pass `userService` when we invoke the `statusAction` in `RestoreOrRevokeAsync`, because we don't know which underlying method is being called. So we either add the `userService` parameter to `RevokeUsersAsync` (which would be redundant) or use this lambda to effectively remove it from `RestoreUsersAsync`.

But yeah... all stuff we shouldn't have to do in any case, really we should get rid of the circular dep! I think it's a good candidate for a "command" per the new CQRS pattern.